modules/dop/services/issue/issue.go

```diff
@@ -657,7 +657,15 @@ func (svc *Issue) UpdateIssue(req apistructs.IssueUpdateRequest) error {
 	}
 
 	if issueModel.Type == apistructs.IssueTypeTask {
-		if err := svc.AfterIssueChildrenUpdate(issueModel.ID); err != nil {
+		currentBelong, err := cache.TryGetState(issueModel.State)
+		if err != nil || currentBelong == nil {
+			return apierrors.ErrUpdateIssue.InternalError(err)
```

Comment:
if err == nil, the error message will empty, I suggest to compose a new error

Reply:
done